superset-frontend/src/dashboard/components/SliceHeaderControls.jsx

```diff
@@ -158,6 +166,8 @@ class SliceHeaderControls extends React.PureComponent {
             </MenuItem>
           )}
 
+          {<MenuItem onClick={this.handleFullSize}>{t(resizeLabel)}</MenuItem>}
```

Comment:
this doesn't need to be wrapped in braces

Reply:
Also, `resizeLabel` shouldn't be wrapped in `t()` again if you've already wrapped the strings in it

Reply:
Changed c0526bd607a84ebc9e32f7d4676af14a06a2624b